boefjes/tests/plugins/test_wappalyzer_normalizer.py

```diff
@@ -6,10 +6,14 @@ def test_page_analyzer_normalizer(normalizer_runner):
     meta = NormalizerMeta.model_validate_json(get_dummy_data("body-page-analysis-normalize.json"))
     output = normalizer_runner.run(meta, get_dummy_data("download_page_analysis.raw"))
 
+    assert output.observations
     results = output.observations[0].results
-    assert len(results) == 6
+    assert len(results) == 14
     assert {o.primary_key for o in results if o.object_type == "Software"} == {
+        "Software|BootstrapCDN|3.3.7|",
+        "Software|Bootstrap|3.3.7|cpe:2.3:a:getbootstrap:bootstrap:*:*:*:*:*:*:*:*",
+        "Software|cdnjs||",
         "Software|jQuery Migrate|1.0.0|",
-        "Software|jQuery|3.6.0|",
-        "Software|Bootstrap|3.3.7|",
+        "Software|jQuery|2.3|cpe:2.3:a:jquery:jquery:*:*:*:*:*:*:*:*",
```

Comment:
I think this detection is invalid, or more specifically. there is no version 2.3 of jquery in the input fie. 2.3 refers to the version of CPU uri.